src/Components/Auth/LoginForm.js

```diff
@@ -1,49 +1,128 @@
 import React, { useState } from 'react';
+import { useFormik } from 'formik';
+import {
+  Alert,
+  AlertTitle,
+  Button,
+  Container,
+  FormControl,
+  Input,
+  InputAdornment,
+  InputLabel,
+} from '@mui/material';
+import EmailOutlinedIcon from '@mui/icons-material/EmailOutlined';
+import VisibilityOutlinedIcon from '@mui/icons-material/VisibilityOutlined';
 import '../FormStyles.css';
 
+const validate = (values) => {
+  let errors = {};
+
+  if (!values.password) {
+    errors.password = 'required!';
+  } else if (values.password.length < 6) {
+    errors.password = 'at least 6 characters';
+  } else if (!/[\d]{1}/g.test(values.password)) {
```

Comment:
what this regex is for? please move it to a constant with a representative name, it can be placed on Utils/Validations. Other thing that you can do is create a new function in Utils/Validations to test if a password is valid
```
const containsOneNumber = (value) => /[\d]{1}/g.test(value);
```
then you use it:
```
else if (!containsOneNumber(values.password))
```